apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -106,6 +106,7 @@ export class TeacherFeedback extends Component {
     visible: PropTypes.bool.isRequired,
     //Provided by Redux
     viewAs: PropTypes.oneOf(['Teacher', 'Student']).isRequired,
+    serverScriptId: PropTypes.number,
     serverLevelId: PropTypes.number,
     serverScriptLevelId: PropTypes.number,
```

Comment:
Will `severScriptLevelId` still be needed for anything?

Reply:
Good question Erin! My plan was to do that in step 5 in the doc, when we stop writing to script_level_id on the server. The idea would be to keep step 4 as small as possible, so that it is easy to reason about and/or revert if something goes wrong.

Reply:
Documentation and small, atomic PRs! ![](https://media.giphy.com/media/kbuQOkATEo6VW/giphy.gif)

Reply:
Thanks for the kudos, @Erin007 ! 😄